src/Instrumentation/PDO/composer.json

```diff
@@ -8,7 +8,8 @@
   "license": "Apache-2.0",
   "minimum-stability": "dev",
   "require": {
-    "php": "^8.2",
+    "php": "^8.0",
```

Comment:
PDO requires at least 8.2 as since this version hooking into built-in functions is possible